msteams-platform/graph-api/meeting-transcripts/overview-transcripts.md

```diff
@@ -57,3 +57,4 @@ For more information about how your app can know when the meetings ends, see [su
 - [Apps for Teams meetings and calls](../../apps-in-teams-meetings/teams-apps-in-meetings.md)
 - [Meeting app APIs](../../apps-in-teams-meetings/meeting-apps-apis.md)
 - [App manifest schema for Teams](../../resources/schema/manifest-schema.md)
+- [OnlineMeeting resource type](/graph/api/resources/onlinemeeting)
```

Comment:
The link text is vague. Describe the link in natural language of the user instead of using jargon. Base it on "_Contains information about a meeting, including the URL used to join a meeting, the attendees list, and the description._" description in the linked article.